sdk/core/azure-core/inc/azure/core/nullable.hpp

```diff
@@ -59,7 +59,7 @@ template <class T> class Nullable final {
    * @param other Another `%Nullable` instance to copy.
    */
   Nullable(const Nullable& other) noexcept(std::is_nothrow_copy_constructible<T>::value)
-      : m_hasValue(other.m_hasValue)
+      : m_disengaged{}, m_hasValue(other.m_hasValue)
```

Comment:
Just confirming, @BillyONeal, @CaseyCarter were these just a minor oversight that got left out from https://github.com/Azure/azure-sdk-for-cpp/pull/204 or is there some intentional reason for not having these here, that I might be missing?

Reply:
I assume these were intentionally omitted since they're not needed for correctness. That said, they don't hurt anything or even contribute any code to the resulting program so it's fine to add them.